tests/abi/enc/CMakeLists.txt

```diff
@@ -21,4 +21,4 @@ add_enclave(
 
 enclave_include_directories(abi_enc PRIVATE ${CMAKE_CURRENT_BINARY_DIR}
                             ${CMAKE_CURRENT_SOURCE_DIR})
-enclave_link_libraries(abi_enc oelibc)
+enclave_link_libraries(abi_enc oelibcxx)
```

Comment:
If this works, it would be good to add the `CXX` to the `add_enclave` function above instead this.

Reply:
Sure. What is the difference between the two?

Reply:
It does the same thing. The idea is that we don't need to manually specify the `oelibc` (always add) or `oelibcxx` (add when `CXX` is supplied) with `add_enclave`.

Reply:
Ah Ok. Done.